src/multipa/main.py

```diff
@@ -485,6 +507,14 @@ def main_cli():
         remove_columns=full_valid_data.column_names,
         num_proc=args.num_proc
     )
+
+    # Check that the post-tokenization data is valid
+    print("Checking that the post-tokenization data is valid...")
+    full_train_data = full_train_data.filter(is_valid_post_tokenization)
+    full_valid_data = full_valid_data.filter(is_valid_post_tokenization)
```

Comment:
Again, let's see if this works with multiprocessing...
```suggestion
    full_train_data = full_train_data.filter(is_valid_post_tokenization, args.num_proc)
    full_valid_data = full_valid_data.filter(is_valid_post_tokenization, args.num_proc)
```